src/EditorFeatures/CSharpTest/Completion/CompletionProviders/ExplicitInterfaceMemberCompletionProviderTests.cs

```diff
@@ -157,5 +220,189 @@ class Bar : I1
+    }
+}
+
+interface IBar : IGoo
+{
+     void IGoo.$$
+}";
+
+            await VerifyItemIsAbsentAsync(markup, "Goo");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task NotInaccessibleMember_01()
+        {
+            var markup =
+@"<Workspace>
+    <Project Language=""C#"" AssemblyName=""Assembly1"" CommonReferences=""true"">
+        <ProjectReference>Assembly2</ProjectReference>
+        <Document FilePath=""Test1.cs"">
+<![CDATA[
+class Bar : IGoo
+{
+     void IGoo.$$
+}
+]]>
+        </Document>
+    </Project>
+    <Project Language=""C#"" AssemblyName=""Assembly2"" CommonReferences=""true"" LanguageVersion=""Preview"">
+        <Document FilePath=""Test2.cs"">
+interface IGoo
+{
+    internal void Goo1() {}
+    internal int Prop1 { get => 0; }
+    protected void Goo2() {}
+    protected int Prop2 { get => 0; }
+}
+        </Document>
+    </Project>
+</Workspace>";
+
+            await VerifyItemExistsAsync(markup, "Goo1()"); // PROTOTYPE(DefaultInterfaceImplementation): Should use VerifyItemIsAbsentAsync instead after test infrastructure is changed to actually build two distinct projects from the XML 
+            await VerifyItemExistsAsync(markup, "Prop1"); // PROTOTYPE(DefaultInterfaceImplementation): Should use VerifyItemIsAbsentAsync instead after test infrastructure is changed to actually build two distinct projects from the XML 
+            await VerifyItemExistsAsync(markup, "Goo2()");
+            await VerifyItemExistsAsync(markup, "Prop2");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task NotInaccessibleMember_02()
+        {
+            var markup =
+@"<Workspace>
+    <Project Language=""C#"" AssemblyName=""Assembly1"" CommonReferences=""true"">
+        <ProjectReference>Assembly2</ProjectReference>
+        <Document FilePath=""Test1.cs"">
+<![CDATA[
+interface IBar : IGoo
+{
+     void IGoo.$$
+}
+]]>
+        </Document>
+    </Project>
+    <Project Language=""C#"" AssemblyName=""Assembly2"" CommonReferences=""true"" LanguageVersion=""Preview"">
+        <Document FilePath=""Test2.cs"">
+interface IGoo
+{
+    internal void Goo1() {}
+    internal int Prop1 { get => 0; }
+    protected void Goo2() {}
+    protected int Prop2 { get => 0; }
+}
+        </Document>
+    </Project>
+</Workspace>";
+
+            await VerifyItemExistsAsync(markup, "Goo1()"); // PROTOTYPE(DefaultInterfaceImplementation): Should use VerifyItemIsAbsentAsync instead after test infrastructure is changed to actually build two distinct projects from the XML 
+            await VerifyItemExistsAsync(markup, "Prop1"); // PROTOTYPE(DefaultInterfaceImplementation): Should use VerifyItemIsAbsentAsync instead after test infrastructure is changed to actually build two distinct projects from the XML 
+            await VerifyItemExistsAsync(markup, "Goo2()");
+            await VerifyItemExistsAsync(markup, "Prop2");
+        }
```

Comment:
>} [](start = 8, length = 1)

It looks like we also don't complete keywords (like `private`) in interfaces. Could be a follow-up PR.